pytest_mpl/plugin.py

```diff
@@ -128,11 +132,19 @@ def wrapper(*args, **kwargs):
         item.obj = figure_interceptor(plugin, item.obj)
 
 
-def pytest_report_header(config, startdir):
```

Comment:
Can we just remove all the arguments as none of them are used in the hook?

Reply:
I can try that.

Yes, I opened this PR because it was failing in astropy devinfra job.